### ISSUE TITLE
escape quotes in Strings when enforcing a new quote

### ISSUE THREAD

**Issue by Janther** [prettier-solidity/prettier-plugin-solidity#141]
Currently this:
```Solidity
str = '"Be yourself; everyone else is already taken." -Oscar Wilde';
```
Will become this:
```Solidity
str = ""Be yourself; everyone else is already taken." -Oscar Wilde";
```

**Comment by mattiaerre** [comment]
@Janther I'm not too sure: is this a bug or an enhancement?

**Comment by Janther** [comment]
Bug
Will be fixed once #144 passes all the tests and gets merged

**Comment by mattiaerre** [comment]
closing thanks to #144
